cli/mender-artifact/copy.go

```diff
@@ -37,7 +37,7 @@ func Cat(c *cli.Context) (err error) {
 	if !isimg.MatchString(c.Args().First()) {
 		return cli.NewExitError("The input image does not seem to be a valid image", 1)
 	}
-	r, err := NewPartitionFile(comp, c.Args().First(), c.String("key"))
+	r, err := virtualPartitionFile.Open(comp, c.Args().First(), c.String("key"))
```

Comment:
Changed the API so that it is looks more like a regular file (better semantics imo)

Reply:
Not an expert in golang (yet!), but I thought that any symbol to be exported in a file needs to start with capital letter. I would then expect the constant `virtualPartitioFile` to be `VirtualPartitionFile`.

I also wonder about the reason for such design. I would have expected a NewPartitionFile method returning an object that satisfy VPFile interface, and from there the user can call open/close, etc

Thanks for clarifying!